hazelcast/src/test/java/com/hazelcast/client/config/InvalidConfigurationClientTest.java

```diff
@@ -136,7 +136,7 @@ public void WhenValid_NearCacheInMemoryFormat() {
 
     @Test(expected = InvalidConfigurationException.class)
     public void WhenInvalid_NearCacheInMemoryFormat() {
-        buildConfig("near-cache-in-memory-format", "binaryyy");
+        buildConfig("near-cache-in-memory-format", "binary_");
```

Comment:
I'm assuming it doesn't really matter what the string is as long as it isn't `binary`. I can drop any particular change/ignore the item, but if it doesn't matter, ...